spec/models/merchant_spec.rb

```diff
@@ -38,69 +38,69 @@
       expect(Merchant.all).to eq(original)
```

Comment:
It looks like line 34 is an extra line that could be deleted

Reply:
This isn't from my PR or a method I wrote. Everything I was doing was most utilizing Items.